templates/ts-apollo-mongo-apache-kafka-backend/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3'
+services:
+
+  zookeeper:
+    image: wurstmeister/zookeeper
+    ports:
+      - 2181:2181
+
+  kafka:
+    image: wurstmeister/kafka
```

Comment:
Where this is comming from?

`image: apache-kafka/broker:2.5.0` 

We should use kafka from strimzi or if you do not like strimzi then just use official sources. 
This is some guy there. Just swap it from docker-compose from kafka website.

Reply:
I chose this because it is by far the most used Kafka image on Docker (> 10M pulls)

https://hub.docker.com/r/wurstmeister/kafka

I could not find a better alternative.
```
$ docker pull apache-kafka/broker
Using default tag: latest
Error response from daemon: pull access denied for apache-kafka/broker, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

https://hub.docker.com/r/strimzi/kafka

strimzi/kafka has no documentation on how to set it up - I think it is only intended for direct use with the Strimzi Operator.

We _could_ figure out how to use it by looking at the operator documentation, but this just makes it almost impossible for a regular user to make changes without also having to do the same.